python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -508,9 +514,16 @@ def run(
         self._add_objects(wcsf, inputCatalogRefs, sourceIndices, extensionInfo, usedColumns)
         self._add_ref_objects(wcsf, refObjects, refCovariance, extensionInfo)
 
+        # There must be at least as many sources per visit as the number of
+        # free parameters in the per-visit mapping. Set minFitExposures to be
+        # the number of free parameters, so that visits with fewer visits are
+        # dropped.
+        nCoeffVisitModel = int(((2 * self.config.exposurePolyOrder + 3) ** 2 - 1) / 8)
```

Comment:
Could you briefly comment on how this calculation comes about? Even better would be to make this a utility function and comment it as a docstring, since you use this calculation at least twice in this code.

Reply:
Ok, I decided to make utility functions for this and for the inverse operation.